scripts/ci-e2e.sh

```diff
@@ -36,7 +44,17 @@ clone_repo "${M3_DEV_ENV_REPO}" "${M3_DEV_ENV_BRANCH}" "${M3_DEV_ENV_PATH}"
 cp "${REPO_ROOT}"/hack/e2e/environment.sh "${M3_DEV_ENV_PATH}/config_${USER}.sh"
 
 pushd ${M3_DEV_ENV_PATH}
-make
+# Golang needs to be installed before cloning the needed repos to the Go source directory.
+make install_requirements configure_host 
+# shellcheck disable=SC1091
+source lib/common.sh
```

Comment:
I wonder if this is needed, we do source it in prev. step (01_* and 02_* scripts), no?

Reply:
Yes, it is needed, otherwise it will fail. I have tested it.